create_project.yml

```diff
@@ -17,3 +20,7 @@
         description: "{{ project_description|default(project_display_name)|default(project_name) }}"
         display_name: "{{ project_display_name|default(project_name) }}"
         state: present
+      register: new_project
+      until: new_project.result is defined and new_project.result.status is success
+      retries: 30
+      delay: 10
```

Comment:
so that's all we need to do to make it synchronous?

Reply:
then we could use the same pattern to make jobs wait for edxapp to be up?

Reply:
You read my mind, it's already planned :wink: I'll dedicated tasks to run jobs. But still, we will need to improve jobs ordering/dependency.